NybergViktor/Celemo_FrontEnd
# 73 create a review on user

# Hello

- [x] Tommy

## whats done:
* major redesign on public profile page (css and html)

![Screenshot 2024-05-15 at 19 43 40](https://github.com/NybergViktor/Celemo_FrontEnd/assets/144717637/9148e8a0-b1ba-48d4-a7bf-1e01fd9e713f)

And a:

* New function that lets users submit new reviews on other users.

* clickable button on public profile (review user) that leads to a new page where you can review user.

![Screenshot 2024-05-15 at 19 43 46](https://github.com/NybergViktor/Celemo_FrontEnd/assets/144717637/14997d89-9fc9-4609-b87e-5f5ba270606f)

* when you have succesfully submitted a review, you will get redirected to a new page called "/return". Through this you can find back to startpage.

![Screenshot 2024-05-15 at 19 46 21](https://github.com/NybergViktor/Celemo_FrontEnd/assets/144717637/34358d11-2e4c-4586-8f90-f8c12fe4e571)


* your new review should look like this in MongoDB:

![Screenshot 2024-05-15 at 19 44 44](https://github.com/NybergViktor/Celemo_FrontEnd/assets/144717637/b95094ab-adca-4a49-84e1-40e6d35be7fd)



# What to test

* log in, click into an auction, click on the sellers @, click on review user, set grade and type review text, submit
* check database
* check that id:s are correct

#note -> don't review on your own auctions

# Let me know if you find any errors

--

Tommy needs to fix pubContext before merging this PR (he knows).

--


Closes #42 
Closes #73
